expression-language.md

```diff
@@ -111,7 +110,7 @@ lowercase-char ::= [a-z]
 value-identifier ::= ( lowercase-char | digit ) ( lowercase-char | digit )*
 ```
 
-CESQL defines 3 different literal kinds: integer numbers, `true` or `false` booleans and `''` or `""` delimited strings.
+CESQL defines 3 different literal kinds: integer numbers, `true` or `false` booleans, and `''` or `""` delimited strings.
```

Comment:
I'm quite sure you're better than me with english :), but is the comma here required?

Reply:
https://en.wikipedia.org/wiki/Serial_comma 

Style guides vary, but note that we already have it on 54, 131, 184, 268. It's usually the less-ambiguous choice.